.github/workflows/ci.yaml

```diff
@@ -49,6 +35,17 @@ jobs:
           poetry env use "${{ matrix.python-version }}"
           poetry install
 
+      - name: Downgrade soundfile version on macOS x86_64
+        # soundfile>=0.12.0 prepackages libsndfile, but the prepackaged version gives
+        # different results than the system version for macOS running x86_64
+        # (which is what GitHub Actions uses). As such we have to downgrade to
+        # soundfile==0.11.0 and use the system libsndfile.
+        run: |
+          if [ "$RUNNER_OS" == "macOS" ] && [ "$(uname -m)" == "x86_64" ]; then
+            poetry add soundfile==0.11.0
+          fi
+        shell: bash
```

Comment:
Does this mean that the MacOs server had `libsndfile` pre-installed all along?

Reply:
Yea, looks like it. It is installed in our macOS environment at least. We cannot be sure if it breaks in the future, but hopefully `soundfile` will include another `libsndfile` binary for Darwin x86_64 in the future that gives the same results as on Silicon and Ubuntu and we won't have to downgrade the version in the workflow.